test/e2e/gateway/gatewayapi/conformance_test.go

```diff
@@ -23,15 +23,6 @@ var clusterName = Kuma1
 var minNodePort = 30080
 var maxNodePort = 30089
 
-const gatewayClass = `
```

Comment:
this seems unrelated, the PR is only about `.Capabilities.APIVersions` not being right

Reply:
the problem was that the GatewayClass wasn't being created at install time, which is fixed now. so we no longer need to create this "default kuma Gateway" GatewayClass in e2e tests, it just exists